benchmarking/smartseq2/merge_picard_metrics/merge_picard_mets.py

```diff
@@ -0,0 +1,52 @@
+from crimson import picard
+import pandas as pd
+import numpy as np
+from google.cloud import storage
+import json
+from os.path import basename
+import sys
+import requests
+
+## set up auth
+client = storage.Client()
+bucket=client.get_bucket('broad-dsde-mint-dev-cromwell-execution')
+## load cromwell credential
+logins=json.load(open('/usr/secrets/broad-dsde-mint-dev-cromwell.json'))
+
+## input json has uuid, run name and value to parse.
+uuid=sys.argv[1]
+met_name=sys.argv[2]
+output_name=sys.argv[3]
+##meta_url
+metadata_url="https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1/"+uuid+"/metadata?expandSubWorkflows=false"
```

Comment:
Again, I would encourage having this script take a list of files rather than a uuid. The merge WDL could take an Array[File] and just pass it to this script.

The user would be expected to run both ss2 and the merge WDL as subworkflows, so that it's easy to pass the Array[File] of outputs from ss2 to the merge WDL.

Having to fetch and parse the metadata in the script brings complications that make it harder to productionize (url is different for dev, staging, etc, and we have to worry about security of the username and password), so it will really help us to remove the need for that step.